index.html

```diff
@@ -40,6 +51,7 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
           <h2>My Recent Works</h2>
           <hr>
         </hxr>
+
```

Comment:
- [ ] According to the project requirement you should stick to the design like color, font, margin, etc. please use `#091E42` color on your `h2` heading and there is a line that takes the rest of the page. please increase the width of the line. 😆 
### Your design
![your](https://user-images.githubusercontent.com/81727988/199893356-0ed62cb5-d0c5-4a14-9a04-12242768e850.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/199893437-0bd16407-07d5-4241-8947-9c09712faf5f.png)